### ISSUE TITLE
Facebook API login not working

### ISSUE THREAD

**Issue by leftys** [MLstate/opalang#190]
I tried to implement FB login using the Opa's facebook api. I cloned the example code https://github.com/winbomb/opapackt/blob/master/liveroom/src/login.opa , but it doesn't work. I can get to the point where FB redirects to ``http://localhost:8080/connect?code=something`` after authentication, but FBA.get_token_raw(data, redirect) returns ``opa_data_error [OPA]Error in data``.

Maybe the FB api changed in the meantime?

**Comment by hbbio** [comment]
Yes, FB API changed faster that we could track. You're welcome to submit a PR if you have a fix!
